build.gradle

```diff
@@ -266,6 +266,47 @@ subprojects {
                         artifact javadocJar
                         pom.withXml(pomXmlModifier)
                     }
+                    pom {  // https://central.sonatype.org/pages/requirements.html
+                        name = "${project.name}"
+                        afterEvaluate { project ->
+                            description = "Digdag"
```

Comment:
Does this need to be in `afterEvaluate`? Simple configuration looks to work (e.g. https://github.com/komamitsu/fluency/blob/master/build.gradle#L107)

Reply:
I refer embulk's build.gradle.
In embulk, description is described in each subproject and need to evaluate later.
Of course it is not required in current description "Digdag", anyway no impact to publish.
So I keep it for a while.
https://github.com/embulk/embulk/blob/master/build.gradle#L159-L162